python/ql/lib/semmle/python/dataflow/new/internal/FlowSummaryImpl.qll

```diff
@@ -12,13 +12,16 @@ private import DataFlowImplSpecific::Public
 module Input implements InputSig<DataFlowImplSpecific::PythonDataFlow> {
   class SummarizedCallableBase = string;
 
-  ArgumentPosition callbackSelfParameterPosition() { none() }
+  ArgumentPosition callbackSelfParameterPosition() { result.isLambdaSelf() }
```

Comment:
Yes, we [pulled that out](https://github.com/github/codeql/pull/14944/commits/78c484faab3087eec41779f92572a1cb6d4cd663) since the PR was complicated enough already :-)
Also, the type tracker got a little big...